pages/common/nano.md

```diff
@@ -3,22 +3,26 @@
 > Simple, easy to use command-line text editor. An enhanced, free Pico clone.
 > More information: <https://nano-editor.org>.
 
+- Open a new file in nano:
+
+`nano`
+
 - Open a specific file:
 
 `nano {{path/to/file}}`
 
-- Open a file positioning the cursor at the specified line and column:
+- Open a specific file, positioning the cursor at the specified line and column:
 
 `nano +{{line}},{{column}} {{path/to/file}}`
 
-- Enable smooth scrolling:
+- Open a specific file and enable smooth scrolling:
 
-`nano -S {{filename}}`
+`nano --softwrap {{path/to/file}}`
 
-- Indent new lines to the previous lines' indentation:
+- Open a specific file and indent new lines to the previous lines' indentation:
 
-`nano -i {{filename}}`
+`nano --autoindent {{path/to/file}}`
 
-- Before modification, backup separately as `{{current_file_name}}~`:
+- Open nano and create a backup file (`file~`) if edits are saved:
```

Comment:
I think the old example description is better.

Reply:
I prefer the new one for following reasons:
* "Backup" isn't very specific. To me this could be some file fragments hidden in a temporary system folder or maybe saved anywhere in any other form. `Backup file` implies to me that it copies the unsaved file like it is.
* `file~` directly tells you how the backup file is named, so you don't have to guess how you could find it. And since there is only `file~` without a path it kind of shows that it's in the same directory as the original file.

Reply:
pinging @sbrl

Reply:
I see. What about this?
```suggestion
- Create a backup file (`file~`) when saving edits:
```

Reply:
I think it needs the `open nano` bit then fine by me

Reply:
```suggestion
- Open nano and create a backup file (`file~`) when saving edits:
```